src/Raven.Server/NotificationCenter/Handlers/ServerNotificationCenterHandler.cs

```diff
@@ -42,12 +46,13 @@ public async Task Get()
                     {
                         var action = OperationChanged.Create(null, operation.Id, operation.Description, operation.State, operation.Killable);
 
+                        if (TrafficWatchManager.HasRegisteredClients)
+                            AddStringToHttpContext(writer.ToString(), TrafficWatchChangeType.Notifications);
```

Comment:
we need to write the `action.ToJson()` here since we reset the ctx in `writer.WriteToWebSocket()`

Reply:
Done